### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by Anntoin**

### CODE REVIEW

File: i3session [hunk 1]
```diff
@@ -7,6 +7,7 @@ import pickle
 import subprocess
 import logging
 from time import sleep
+from xdg.BaseDirectory import *
```
Comment (joepestro) [inline i3session:10]: Does this add a new dependency? Add `PyXDG` to readme?

Comment (Anntoin) [inline i3session:10]: It does, I've updated the readme there now

**Comment by joepestro** [comment]
Thanks for the first pull request on this project! Merging into master.

**Comment by Anntoin** [comment]
No problem, thanks for getting it going :)

**Pull request merged**